mapwfs.c

```diff
@@ -4063,6 +4067,7 @@ int msWFSGetPropertyValue(mapObj *map, wfsParamsObj *paramsObj, cgiRequestObj *r
   char* pszGMLGroups = NULL;
   char* pszGMLIncludeItems = NULL;
   char* pszGMLGeometries = NULL;
+  char* papszWFSDefaultMatchCase = NULL;
```

Comment:
Use name papszWFSDefaultMatchCases so it looks the same as others (e.g. pszGMLGroups, pszGMLIncludeItems etc.)